nft-api/minting.md

```diff
@@ -7,35 +7,38 @@ authors:
     email: artur@licenserocks.de
     avatar: ../static/artur.jpeg
     link: https://twitter.com/ArtiChmaro
+  - name: Adam Leszczyński
+    email: adam.leszczynsky@gmail.com
+    link: https://alsd.app
+    avatar: https://avatars.githubusercontent.com/u/20825919
 ---
 # Mint new NFTs
 
 In this section you will learn how to use our endpoints for minting new NFT tokens.
-Forget about deploying own smart contracts.
+It is crucial to remain the order of the requests in the following documentation.
+## 1. Batch mint
 
-## Batch mint
-
-Use below endpoint in order to create multiple tokens in your instance.
-Send array with one element if you want to create just single token.
+Use below endpoint in order to create multiple draft NFTs in your instance.
+Send array with one element if you want to create just single draft NFT.
 
 ```
 POST https://{INSTANCE_URL}/api/public/batchMint
 ```
 
 !!!warning
-this request must contain Authorization header with your token
+This request must contain [Authorization](/authorization.md) header with your token.
 !!!
 
 ```json Payload (application/json)
 [
   {
-  "title": "Crypto Punk",
-  "amount": 1,
-  "price": 9999.5,
-  "currency": "EUR",
-  "useCaseId": 1,
-  "biddingEnabled": false,
-  "status": "draft"
+    "title": "Crypto Punk",
+    "amount": 1,
+    "price": 9999.5,
+    "currency": "EUR",
+    "collectionTemplateId": 1,
```

Comment:
would be awesome if you can explain what collection template is and where to get ID

Reply:
Yeah, that would be cool